### PR TITLE
add isStringNumeric to library

### PR DESCRIPTION
**Pull request by replay**
helper function to check whether all characters in a string are numeric

### CODE REVIEW

File: ascii.libsonnet [hunk 1]
```diff
@@ -26,4 +26,13 @@ local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
...
   isNumber(c): std.isNumber(c) || (cp(c) >= 48 && cp(c) < 58),
+
+  '#isStringNumeric':: d.fn(
+    '`isStringNumeric` reports whether string `s` consists only of numeric characters.',
+    [d.arg('str', d.T.string)]
+  ),
+  isStringNumeric(str): std.all([
+    self.isNumber(c)
+    for c in std.stringChars(str)
+  ]),
```
Comment (Duologic) [inline ascii.libsonnet:37]: Using `std.map` would be a bit more idiomatic:
```suggestion
  isStringNumeric(str): std.all(std.map(self.isNumber, std.stringChars(str)),
```

**Review by Duologic — APPROVED** [review]

File: ascii.libsonnet [hunk 2]
```diff
@@ -26,4 +26,10 @@ local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
     [d.arg('c', d.T.string)]
   ),
   isNumber(c): std.isNumber(c) || (cp(c) >= 48 && cp(c) < 58),
+
+  '#isStringNumeric':: d.fn(
+    '`isStringNumeric` reports whether string `s` consists only of numeric characters.',
+    [d.arg('str', d.T.string)]
+  ),
+  isStringNumeric(str): std.all(std.map(self.isNumber, std.stringChars(str)),
```
Comment (Duologic) [inline ascii.libsonnet:34]:
```suggestion
  isStringNumeric(str): std.all(std.map(self.isNumber, std.stringChars(str))),
```
I think one parentheses is missing

**Comment by replay** [comment]
ping @Duologic , do you think we can merge this?

**Review by Duologic — APPROVED** [review]

**Comment by replay** [comment]
@Duologic sorry to ping again, but I can't merge this myself :) It says:

> Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

**Comment by Duologic** [comment]
No worries, I approved while on my phone, couldn't fix the failing test there. Just fixed it, merging now.

**Pull request merged**